acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/AcceptanceTestBase.java

```diff
@@ -98,6 +108,55 @@ protected AcceptanceTestBase() {
     permissionedNodeBuilder = new PermissionedNodeBuilder();
   }
 
+  static {
+    System.setProperty("log4j2.isThreadContextMapInheritable", "true");
+  }
+
+  @Rule public final TestName name = new TestName();
+
+  @Rule
+  public TestWatcher log_eraser =
+      new TestWatcher() {
+
+        @Override
+        protected void starting(final Description description) {
+          ThreadContext.put("test", description.getMethodName());
+          ThreadContext.put("class", description.getClassName());
+          Thread.currentThread()
```

Comment:
You could put the error string into a local String variable and use Log4j lazy variable substitution.
e.g.
```
final String errorMessage = " "Uncaught exception in thread \"{}\"";
```

Reply:
Discussed and done.